package.json

```diff
@@ -3,7 +3,7 @@
   "name": "@substrate/api-sidecar",
   "description": "REST service that makes it easy to interact with blockchain nodes built using Substrate's FRAME framework.",
   "homepage": "https://github.com/paritytech/substrate-api-sidecar#readme",
-  "author": "Parity Technologies <admin@parity.io>",
+  "author": "Logion Team",
```

Comment:
I would turn this into "contributors" (see [here](https://docs.npmjs.com/cli/v6/configuring-npm/package-json#people-fields-author-contributors)) and mention both Parity and us.

Reply:
Done.